### ISSUE TITLE
`lazygit -v` is basically empty

### ISSUE THREAD

**Issue by c02y** [jesseduffield/lazygit#1162]
**Describe the bug**
When I install lazygit using `go get`, the result of `lazygit -v` is:
```
commit=, build date=, build source=unknown, version=unversioned, os=linux, arch=amd64
```
So I don't know any useful info when I try to file the bug report

**To Reproduce**

**Expected behavior**
`lazygit -v` should provide useful version info

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]  Manjaro-Linux
 - Lazygit Version [e.g. v0.1.45] installed from `go get`, no version info
 - The last commit id if you built project from sources (run : ```git-rev parse HEAD```)

**Additional context**

**Comment by jesseduffield** [comment]
@c02y how did you install lazygit? That behaviour occurs when running it directly from source because you need to compile it with e.g. commit SHA, tag, to have that stuff included

**Comment by c02y** [comment]
> Lazygit Version [e.g. v0.1.45] installed from go get, no version info

**Comment by Raingram** [comment]
I just got this same issue, since you deprecated the ubuntu PPA, I thought using `go get github.com/jesseduffield/lazygit` would be the next best method.

So, I installed golang, then used `go get github.com/jesseduffield/lazygit` to install lazygit.
I can only assume this is what you mean when you say, "running it directly from source"?

@jesseduffield are you saying that to have the version info I can't use ` go get` and must clone the repo instead?
(I want to avoid installing a second, and therefore redundant, package manager.)
